pkg/featuregate/features.go

```diff
@@ -16,11 +16,16 @@
 // optionally overridden.
 package featuregate
 
+const (
+	// FeatureCARMv2 is the name of the CARMv2 feature.
+	FeatureCARMv2 = "CARMv2"
```

Comment:
nit:
```suggestion
	CARMv2 = "CARMv2"
```
I'm thinking about reducing the verbosity when reusing this variable in `r.cfg.FeatureGates.IsEnabled`
```go
r.cfg.FeatureGates.IsEnabled(featuregate.CARMv2)
```
Thoughts